app/bindings/api/v0/bindings/get_highlights_parameters.rb

```diff
@@ -20,6 +20,9 @@ class GetHighlightsParameters
     # Limits results to the source document container in which the highlight was made.  For openstax_page source_types, this is a versionless book UUID. If this is not specified, results across scopes will be returned, meaning the order of the results will not be meaningful.
     attr_accessor :scope_id
 
+    # One or more sets to load data from; default is \"user:me\"
+    attr_accessor :sets
```

Comment:
i would think a more descriptive name would be better here.  "sets" seems kinda vague.  maybe "user_sets"?